lib/dictionaries/api/parameterizer.js

```diff
@@ -0,0 +1,12 @@
+module.exports = function() {
+  return {
+    default: function(req, res, next) {
+      if(req.params.scope !== 'users' && req.params.scope !== 'accounts') {
+        res.status(400);
+        res.send({'error':'scope does not have a valid value'});
+        return;
```

Comment:
i disagree about always...

if  there were more conditions to be tested i'd rather see code which immediately exited when one failed, rather than persisting that to the end of the function flow with lots of indented elses.

Having said that, these middleware layers shouldn't need to get so complex. The complexity would come from multiple separate middleware layers.